client/orderbook/orderbook.go

```diff
@@ -374,7 +421,9 @@ func (ob *OrderBook) unbook(note *msgjson.UnbookOrderNote, cached bool) error {
 	var oid order.OrderID
 	copy(oid[:], note.OrderID)
 
+	ob.ordersMtx.Lock()
 	order, ok := ob.orders[oid]
+	ob.ordersMtx.Unlock()
```

Comment:
Hmm, another surprising one.  OrderBook use patterns must have been preventing concurrent access to the map.

Reply:
I do believe that's the case, but not behavior I wanted to rely upon.